python/graphstorm/eval/eval_func.py

```diff
@@ -275,12 +327,16 @@ def compute_acc_lp(pos_score, neg_score):
 
     return {"lp_fast_score": lp_score}
 
-
 def compute_roc_auc(y_preds, y_targets, weights=None):
     """ compute ROC's auc score
     """
     y_true = y_targets.cpu().numpy()
     y_pred = y_preds.cpu().numpy()
+
+    # check for binary cases, input 2D and label 1D
+    if len(y_pred.shape) > 1 and y_pred.shape[1] == 2 and len(y_true.shape) == 1:
```

Comment:
I think roc_auc_score has already handled this case.

Reply:
Sklearn does not handle this case. We need to give the required format for binary cases.

Reply:
I also modify this condition, including the `y_true.shape` is `(n, 1)` case.